### PR TITLE
Add governance document and design folder

### PR DESCRIPTION
**Pull request by johnSchnake**
Signed-off-by: John Schnake <jschnake@vmware.com>

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+## Overview
+
+Sonobuoy, an open source project, is committed to building an open, inclusive, productive and self-governing open source community focused on building a high quality tool that enables users to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes. The community is governed by this document with the goal of defining how community should work together to achieve this goal.
+
+## Code Repositories
+
+The following code repositories are governed by Sonobuoy community and maintained under the vmware-tanzu\sonobuoy organization.
+
+Sonobuoy: Main Sonobuoy codebase
+Sonobuoy Plugins: This repository contains Sonobuoy plugins
```
Comment (jonasrosland) [inline GOVERNANCE.md:14]: These should be bullet points for formatting

Comment (jonasrosland) [reply GOVERNANCE.md:14]: And please link to the repos themselves

File: design/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Sonobuoy Design Documents
+
+This folder is for proposal documents to large features that require more collaboration than just a GitHub issue.
+
+See the [governance] document for more details.
+
+[governance]: https://github.com/vmware-tanzu/sonobuoy/blob/main/Governance.md
```
Comment (jonasrosland) [inline design/README.md:7]: Please make sure the filename is correctly capitalized

**Review by jonasrosland — CHANGES_REQUESTED** [review]
Just 2 minor changes, otherwise this looks good!

**Comment by johnSchnake** [comment]
Fixed based on feedback. Merging after green tests.

**Review by jonasrosland — APPROVED** [review]

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Sonobuoy Governance
+
+This document defines the project governance for Sonobuoy.
+
+## Overview
+
+Sonobuoy, an open source project, is committed to building an open, inclusive, productive and self-governing open source community focused on building a high quality tool that enables users to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes. The community is governed by this document with the goal of defining how community should work together to achieve this goal.
```
Comment (eleanor-millman) [inline GOVERNANCE.md:7]: Need to delete "that enables users to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volume". Instead, maybe say "that enables users to check the conformance of their Kubernetes clusters as well as perform automated cluster validations."

**Pull request merged**